CMakeLists.txt

```diff
@@ -56,21 +56,22 @@ macro(add_flag flag)
   add_cflag(${flag})
   add_cxxflag(${flag})
 endmacro()
-  
-# Set standard version for compiler.
-add_cflag("-std=c99")
-add_cxxflag("-std=c++11")
-
-# Error on non-ISO C.
-add_cflag("-pedantic-errors")
 
 option(WARNINGS "Enable additional compiler warnings" ON)
 if(WARNINGS)
+  # Set standard version for compiler.
+  add_cflag("-std=gnu99")
+  add_cxxflag("-std=c++98")
+
```

Comment:
Yes. By default, gcc uses gnu89, which is sufficient for us at the moment (soon it won't be). @Zer0-One will hopefully make the original PR again which reverts this revert.